composer.json

```diff
@@ -19,18 +19,18 @@
         "ext-json": "*",
         "yiisoft/arrays": "^3.0@dev",
         "yiisoft/db": "^3.0@dev",
-        "yiisoft/strings": "^1.0"
+        "yiisoft/log": "^3.0@dev",
+        "yiisoft/log-target-file": "^3.0@dev",
```

Comment:
Are you sure specific target is required for the package?

Reply:
no.